app/models/consultation_response.rb

```diff
@@ -66,4 +67,12 @@ def update_reading_time
     self.save
   end
 
+  def sanitize_html_tags
+    if self.response_text.body
+      safe_list_sanitizer = Rails::Html::SafeListSanitizer.new
```

Comment:
@BALAJI-BALA - I think instead of having a sanitizer, let's have a validator. So if the UI sends us HTML tags we don't accept, let's just give them an error.